src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -217,6 +193,13 @@ class SingleActiveFIMap extends React.Component<
     }
   }
 
+  public componentDidUpdate(prevProps: any) {
+    if (!prevProps.plan && this.props.plan) {
+      /** Page was reloaded, fetch data again */
+      this.fetchData();
+    }
+  }
```

Comment:
Can we add this conditional to check `prevProps.plan` and `this.props.plan` IDs so we can fetch data again if the `plan` changes?